README.md

```diff
@@ -92,4 +92,31 @@ server.stop()
 ```
 
 ## Utils
-...
+### Logging
+You can configurate logs as you want by config.
+
+Example:
+
+```py
+from sc_kpm.common.identifiers import LOGGER
```

Comment:
I think will be better to import logger name as 

`from sc_kpm import LOGGER_NAME`